.github/workflows/audit.yaml

```diff
@@ -0,0 +1,19 @@
+name: Security audit
+on:
+  schedule:
+    - cron: '0 0 * * *'
+  workflow_dispatch:
+jobs:
+  audit:
+    name: Rust dependency audit
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v1
+      - uses: actions-rs/audit-check@v1
```

Comment:
quick note: these are apparently unofficial actions, and this one in particular is one big minified JS file, which we give access to our GH actions token. Maybe we can just run `cargo audit`?